apps/companies/controllers/companies_controller.ts

```diff
@@ -22,9 +23,11 @@ export default class CompaniesController {
     return this.companyService.findAll(data)
   }
 
-  async create({ request }: HttpContext) {
+  async create({ request, bouncer, response }: HttpContext) {
+    await bouncer.with(CompanyPolicy).authorize('store')
     const data = await request.validateUsing(createCompanyValidator)
 
-    return this.companyService.create(data)
+    const company = await this.companyService.create(data)
+    return response.created(company)
```

Comment:
You don't need to use `response.redirect`

Reply:
I use `response.created`